tests/Resources/Common.robot

```diff
@@ -103,4 +103,20 @@ Verify NPM Version
     ...    against an expected version in a given pod/container
     [Arguments]  ${library}  ${expected_version}  ${pod}  ${namespace}  ${container}=""  ${prefix}=""  ${depth}=0
     ${installed_version} =  Run  oc exec -n ${namespace} ${pod} -c ${container} -- npm list --prefix ${prefix} --depth=${depth} | awk -F@ '/${library}/ { print $2}'
-    Should Be Equal  ${installed_version}  ${expected_version}
\ No newline at end of file
+    Should Be Equal  ${installed_version}  ${expected_version}
+Get The Labels Of Build
```

Comment:
Please, consider implementing the same functionality with OpenShiftLibrary

Reply:
+1.